pontoon/contributors/templates/contributors/widgets/notifications_menu.html

```diff
@@ -48,7 +48,15 @@
         {% if description and (description.startswith("Unreviewed suggestions") or notification.verb == "has reviewed suggestions" or notification.verb == "ignore") %}
           <span class="description">{{ description|safe }}</span>
 
-          <time class="timeago" datetime="{{ notification.timestamp }}">{{ notification.timesince() }} ago</time>
+
+          {% if not notification.is_old %}
+            <time class="timeago" datetime="{{ notification.timestamp }}">{{ notification.timesince() }}</time>
+          {% else %}
+            <time class="older-timeago" datetime="{{ notification.timestamp }}">
+              {{ notification.timestamp.strftime('%B') }} {{ notification.timestamp.day }}, {{ notification.timestamp.year }}
```

Comment:
There's a [custom](https://github.com/mozilla/pontoon/blob/main/pontoon/base/templatetags/helpers.py#L163-L176) Jinja2 [filter](https://jinja.palletsprojects.com/en/stable/templates/#filters) we should be able to leverage here:

```suggestion
              {{ notification.timestamp|format_datetime("date") }}
```

Reply:
Interesting, I'll implement it this way